steps/24_categorical_support/step.md

```diff
@@ -0,0 +1,143 @@
+# Adding categorical support to sktime
+
+Contributors: @Abhay-Lejith
+
+## Introduction
+
+### Objectives 
+Allow users to pass data with categorical features to any estimator. If not supported, return an informative error message. Otherwise it should run without errors.
+
+### Current status:
+* Non-numeric dtypes are completely blocked off in datatype checks.
+* https://github.com/sktime/sktime/pull/5886 - This pr removes these checks but has not been merged yet as most estimators do not support categorical and will break without informative error messages.
+* https://github.com/sktime/sktime/pull/6490 - pr adding feature_kind metadata
+
+
+## Description of proposed designs
+
+
+For each design we will have to discuss 8 combinations of cases:
+1. estimator natively supports categorical: yes/no
+2. categorical feature specified by user: yes/no
+3. categorical present in data: yes/no
+    
+## Specifying categorical features:
+- The reason I proposed this is in case the user wants to treat a column with numerical values as categorical. This will not be detected by feature_kind as it relies only on the dtype.
```

Comment:
"i proposed this" - missing context. Try to separate solutions from requirements, and ensure the document contains context before you reference the context.